great_expectations/rule_based_profiler/domain_builder/column_domain_builder.py

```diff
@@ -403,7 +410,7 @@ def _resolve_list_type_property(
         property_value_type: Union[type, Tuple[type, ...]],
         variables: Optional[ParameterContainer] = None,
     ) -> List[type]:
-        property_value: Optional[property_value_type] = getattr(self, property_name, [])
```

Comment:
This was not a valid annotation

Reply:
@Kilo59 Is it because dynamic annotations are disallowed?  Anything we can do other than "nothing"?  Thanks.

Reply:
It's possible we could do something with a `TypeVar` but I need to think about it more.

Reply:
The possible `Tuple` of types complicates the use of a `TypeVar` here.
Without it, I could declare that the return type is the same type as the `property_value_type` type.